### DISCUSSION TITLE
Remove the `max-height: 500px` rule on Issue Textareas, add option to use Monospace font in all textareas

### DISCUSSION THREAD

**Discussion by daiplusplus** [community/community#4094] [Other Feature Feedback, Questions, & Ideas]
* The `max-height: 500px` rule for the `<textarea>` in GitHub's Issues is too small.
  * When I'm editing large posts I need to open my browser's dev-tools to disable that rule so I can have a decently-sized textarea.
  * Why is it `500px`? It's tiny! My primary monitor has over 2000px vertical height, so this just feels like you're punishing users for posting walls-of-text.
* The use of normal body text font in the same `<textarea>` is also a pain when posting or editing large code samples. Using a monospace typeface would be so much easier.
  * I'm not asking for monospace to be made the default, but I'd love a simple toggle option in the editor toolbar to enable/disable monospace font for the entire textarea. This is just a client-side thing: doesn't need any new GitHub-side storage of this setting or preference: I'm not bothered if it gets reset every time the page load (but save it in a cookie or `window.localStorage`, perhaps?)

**Comment by willsmythe** [comment]
Second part just shipped: A new user setting lets you opt into using a fixed-width font in all Markdown-enabled fields, including issue and pull request comments: https://twitter.com/GHchangelog/status/1448033685723353092

**Reply by daiplusplus** [reply]
Thanks - that's appreciated.

But why are textareas capped at 500px?
